### ISSUE TITLE
file upload doesn't work

### ISSUE THREAD

**Issue by xu4wang** [gollum/gollum#944]
I tried to drag and drop a picutre(.jpg) file to the edit box. but the file was not uploaded.
The gollum was installed via gem. version 3.1.1.

My gollum command line parameters can be found below:

```
WangXudeMacBook-Pro:gollum_wiki wangxu$ ps -ef | grep ruby
  501 34361  7174   0  2:40上午 ttys001    0:00.00 grep ruby
  501 34228 34224   0  2:23上午 ttys006    0:08.19 /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby /usr/bin/gollum --host 127.0.0.1 --port 9046 --live-preview --allow-uploads page --page-file-dir wiki --css /Users/wangxu/Documents/gollum_wiki
WangXudeMacBook-Pro:gollum_wiki wangxu$ 

```

**Comment by dometto** [comment]
Drag-and-drop uploading is a feature only in gollum's 'normal' editor, not in live-preview mode (because live-preview uses the ACE editor). I'll make this clear in the wiki. Sorry for the confusion.

**Comment by xu4wang** [comment]
Thanks for the clarification.

If I want to use ACE editor during most of the time, and switch to normal editor for drag and dorp.. Is it possible?

**Comment by xu4wang** [comment]
Thanks @dometto 

I tried create a new page, and it was using normal editor. I drag and drop a file into to editor, some text was inserted automatically:

```
[[wiki/SomedayMaybe/somepic.jpg]]
```

However, when I save the page, the picture is showed.  I checked the file system, the somepic.jpg was not found.  It looks like the file being droped was not uploaded, only a link was added to current page being edited?

Anything wrong with my installation?
